### DISCUSSION TITLE
Floppy

### DISCUSSION THREAD

**Discussion by abcd4455663** [86Box/86Box#4314] [Ideas]
Add unlimited speed for fdd

**Comment by ghost** [comment]
In the plans for 4.2.

**Comment by abcd4455663** [comment]
Why realise 4.2?

**Reply by TC1995** [reply]
It's the development cycle, that's why.

**Reply by abcd4455663** [reply]
Realise date?

**Reply by TC1995** [reply]
No ETA.

**Reply by DDXofficial** [reply]
When it's ready.

Thread locked as OP is a known ban evader.
